controllers/MailController.js

```diff
@@ -0,0 +1,34 @@
+import 'dotenv/config';
+import { sendgrid } from '../helpers';
+
+const { FRONTEND_URL = '/' } = process.env;
+
+/**
+ * @author Olivier
+ * @param {Object} user
+ * @returns {Promise} - Returns a promise
```

Comment:
`@returns {Promise}` gives enough context. No need for a comment beside the statement.

Reply:
@kimotho-njoki eslint complains if I don't provide a description following @returns {Promise}